crates/server/src/authority/catalog.rs

```diff
@@ -646,7 +653,9 @@ async fn send_forwarded_response(
         }
     };
 
-    response_header.set_authentic_data(answers.dnssec_validated());
+    if can_validate_dnssec {
```

Comment:
Just trying to make sense of the setup here, did we take the `checking_disabled` flag into account in the code leading up to this?